cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityTopicOperator.java

```diff
@@ -104,7 +105,7 @@ protected EntityTopicOperator(Reconciliation reconciliation, HasMetadata resourc
 
         // create a default configuration
         this.kafkaBootstrapServers = KafkaResources.bootstrapServiceName(cluster) + ":" + KafkaCluster.REPLICATION_PORT;
-        this.reconciliationIntervalMs = EntityTopicOperatorSpec.DEFAULT_FULL_RECONCILIATION_INTERVAL_SECONDS * 1_000;
+        this.reconciliationIntervalMs = EntityTopicOperatorSpec.DEFAULT_FULL_RECONCILIATION_INTERVAL_MS;
```

Comment:
Maybe it would be a bit cleaner if you set the default only in one place? Either here and then use `if-elseif` to set the value form the API. Or handle everything in the method. Same for the UO.